test/widgets/autocomplete_test.dart

```diff
@@ -77,34 +101,37 @@ void main() {
       await tester.enterText(composeInputFinder, 'hello @user ');
       await tester.enterText(composeInputFinder, 'hello @user t');
       await tester.pumpAndSettle(); // async computation; options appear
+
       // "User Two" and "User Three" appear, but not "User One"
-      check(tester.widgetList(find.text('User One'))).isEmpty();
-      tester.widget(find.text('User Two'));
-      tester.widget(find.text('User Three'));
+      checkUserShown(user1, store, expected: false);
```

Comment:
This version is fine. But for other situations where the shared context is more verbose than just `store, ` and so it's more of a pain to have to pass it each time, there are two solutions we often use:
* The helper function can be local within the test function, so that it sees the same `store` as a local.
* The context like `store` can be a `late` variable that's declared up at the top of a test group, or of the whole test `main`, and that gets initialized afresh by a helper function that each test case calls. Those helpers are often named `prepare` or of the form `prepareFoo`.